lib/player.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../lib/board.rb'
+require_relative '../lib/game_status.rb'
+class Player
+  attr_reader :player
+  def initialize
+    @player = 'X'
+  end
+
+  def Play(game_board, tic_tac_game, game_status, game_player)
+    tic_tac_game.turn(game_board, game_player, tic_tac_game.name1, tic_tac_game.name2, game_status.win_combinations, tic_tac_game) until game_status.game_over?(game_board.board, game_status.win_combinations)
+
+    return "Congratulations #{game_status.winner(game_board.board, tic_tac_game.name1, tic_tac_game.name2, game_status.win_combinations)} , You Won" if game_status.won?(game_board.board, game_status.win_combinations)
+
+    return "#{tic_tac_game.name1},#{tic_tac_game.name2} Game has been a Draw Well Played " if game_status.draw?(game_board.board, game_status.win_combinations)
+  end
```

Comment:
Learn to keep your character length to a max of between 80 and 90. It will help you in the long run.